PocketKit/Sources/PocketKit/MyList/ItemsList/ItemsListViewModel.swift

```diff
@@ -51,13 +51,16 @@ enum ItemsListCell<ItemIdentifier: Hashable>: Hashable {
 }
 
 enum ItemsListFilter: String, Hashable, CaseIterable {
+    case search = "Search"
     case all = "All"
     case tagged = "Tagged"
     case favorites = "Favorites"
     case sortAndFilter = "Sort/Filter"
 
     var image: UIImage? {
         switch self {
+        case .search:
+            return UIImage(systemName: "magnifyingglass")
```

Comment:
let's utilize `ImageAsset` so we're consistent and can modify all the image strings there

Reply:
We want to use `ImageAsset` for a system default image?  I thought `ImageAsset` was used for our custom images that we have, not for system images.

Reply:
Ah now that I'm looking at it I believe we want to use a custom image. The magnifying glass is different. let's clarify with Tais on which one to use

![image](https://user-images.githubusercontent.com/6743397/202545984-44e90eef-487a-46f1-9a66-96df9f64a51a.png)

Reply:
followed up with Tais. I will update the image.

Reply:
great! thanks for clarifying with her